sdk/docs/core/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 ## 1.0.0-preview.4 (Unreleased)
 
 - API breaking changes:
+  - `az_span.h`:
```

Comment:
In this case, I'd name it as `az_span`, not `az_span.h`.

Reply:
The reason why have the header file is because APIs/methods other than just `az_span` also changed (such as the allocator context).

Reply:
In my mind, it is still super closely related to `az_span` itself, but I see the reason to do it either way.

Reply:
We are distributed by source, which makes me lean towards leaving it as is. Folks do `#include <azure/core/az_span.h>`, so I think it is fine to use the same terminology when talking about particular header file changes.

Besides, we also have az_config.h below :)

Reply:
az_config.h is special, because it is the only common thing about 3 constants.
I am fine with az_span.h.